types/onfido-sdk-ui/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
```

Because we support passing HTML element to SDK options via `containerEl` so we need `dom` lib here

```suggestion
        "lib": [
            "es6",
            "dom"
        ],
```